### ISSUE TITLE
Games need quick sleep

### ISSUE THREAD

**Issue by TheMrIron2** [Game-Fusion/CrossBow#3]
The games need fast sleep to maximise their enjoyable potential. This is replacing sleep() [which takes 1 tick, or 50 milliseconds] with os.queueEvent("name") os.pullEvent(), which is an instant sleep, basically. If anyone can implement this into os.startTimer(), that would be fantastic.

**Comment by byte-chan** [comment]
not even close

**Comment by TheMrIron2** [comment]
Yeah... that's not an "instant" sleep but it would be nice if you could include that
